templates/confirmation.html

```diff
@@ -1,8 +1,89 @@
+{% extends 'base.html' %}
 {% block page %}
-{% include 'components/header.html' %}
+    {% include 'components/header.html' %}
     <main>
-        <p>{{event_data['title']}}</p>
 
+        <div class="Confirmation">
+            <h1 class="Confirmation__title">Event Submitted for Approval!</h1>
+
+            <div class="Submission Submission__box">
```

Comment:
Why is this both Submission and Submission__box? Submission is the block-level, Submission__box is an element-level. [Read this, it'll hopefully clear things up.](http://getbem.com/introduction/)